ibis/sql/alchemy.py

```diff
@@ -1172,6 +1181,55 @@ def drop_table(
         except KeyError:  # schemas won't be cached if created with raw_sql
             pass
 
+    def load_data(
+        self,
+        table_name: str,
+        data: pd.DataFrame,
+        database: str = None,
+        if_exists: str = 'fail',
+        **kwargs,
+    ):
+        """
+        Load data from a dataframe to the backend.
+
+        Parameters
+        ----------
+        table_name : string
+        data: pandas.DataFrame
+        database : string, optional
+        if_exists : string, optional, default 'fail'
+            The values available are: {‘fail’, ‘replace’, ‘append’}
+
+        Raises
+        ------
+        NotImplementedError
```

Comment:
huh?

Reply:
it is related to this:

```
           raise NotImplementedError(
                'Loading data to a table from a different database is not '
                'yet implemented'
            )
```

probably the original message is much better. I will use that for the documentation.